src/sentry/dynamic_sampling/rules/utils.py

```diff
@@ -49,6 +49,24 @@ class RuleType(Enum):
 REVERSE_RESERVED_IDS = {value: key for key, value in RESERVED_IDS.items()}
 
 
+SamplingValueType = Literal["sampleRate", "factor", "rollback"]
```

Comment:
What is `rollback`? Does that exist on Relay side as well?

Reply:
Okay that is a mistake on my side, I have no idea why I wrote it. Will remove it.